framework/Dockerfile

```diff
@@ -34,6 +34,30 @@ ENV IDRINTH_API_BENCH_VERSION=$BUILD_VERSION
 ENV IDRINTH_API_BENCH_BUILT=$BUILD_TIME
 
 RUN mkdir -p /usr/local/lib/node_modules/@idrinth/api-bench
+RUN git clone https://github.com/idrinth-api-bench/framework \
```

Comment:
Are you sure you want to clone into a random directory?

Reply:
adjusted url link